brave/brave-core
# Native assets for all preloading networks

Resolves 

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
Old profile:
1. Before testing, make sure your old profile having some custom tokens already in our new preloading networks. (Switch network to Fantom Opera, visit https://www.coingecko.com/en/coins/wrapped-turtlecoin and add coin.)
2. Check every preloading networks, should have native token listed as the first one in wallet portfolio page.

New profile:
1. Check every preloading networks, should have native token listed in wallet portfolio page.

How it should look for example:
<img width="1539" alt="Screen Shot 2022-06-09 at 3 07 20 PM" src="https://user-images.githubusercontent.com/4730197/172953370-f29c1be7-8a85-43a8-8303-4f23314cd048.png">
